plugin/plugin_health_check.go

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package plugin
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/mattermost/mattermost-server/mlog"
+)
+
+const (
+	PLUGIN_HEALTH_CHECK_INTERVAL  = 5 // minutes
```

Comment:
Seems a bit long. I would want to know if my plugin was unhealthy faster than that. Maybe 1 minute? 30 seconds?

Reply:
👍 30 seconds sounds good to me